File: backend/src/main/java/com/QueueEase/backend/dto/AuthResponse.java
package com.QueueEase.backend.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AuthResponse {
    private String accessToken;
    private String role;
    private String email;
    private String userId;
    private String message;
    private boolean success;
    private String firebaseToken;
}

File: mobile/app/src/main/java/edu/alia/queueease/core/network/models/AuthResponse.kt
package edu.alia.queueease.core.network.models

data class AuthResponse(
    val success: Boolean,
    val message: String?,
    val accessToken: String?,
    val userId: String?,
    val role: String?,
    val firebaseToken: String?,
    val email: String?,
    val firstname: String?,
    val lastname: String?
)
